app/controllers/api/v2/permissions_controller.rb

```diff
@@ -25,6 +25,15 @@ def resource_types
         @total = @resource_types.size
         render :resource_types, :layout => 'api/v2/layouts/index_layout'
       end
+
+      api :GET, "/permissions/current_permissions", N_("List all permissions for current user")
+
```

Comment:
Sorry for not pointing this out earlier, somehow I hoped rubocop would complain about it
```suggestion
```